src/video/video.c

```diff
@@ -246,3 +248,10 @@ void video_draw_remap(const surface *src_surface, int x, int y, int remap_offset
     dst.y = y;
     draw_args(src_surface, &dst, remap_offset, remap_rounds, 0, 255, 255, 0, options);
 }
+
+void video_render_text_block(text_object *text) {
```

Comment:
I'm not sure this belongs in video.c

Reply:
The idea here was that the video plugin can decide what to do with a text object.
For instance the N64 side doesn't traverse the letters if the object is not marked dirty. The code in here for video.c (PC) is unchanged from the original call pattern. If there is a better place that would allow the renderer to decide how to handle the text objects, I'm open to suggestions. This really seemed like the correct place. For instance the NULL renderer will just do nothing here too.

Reply:
This is why we discussed the possible pluggable text renderer backend earlier. The plugin backend would decide how to call the video renderer instead.

Reply:
I agree, but maybe we can do something intermediate for now, before we go to the "solves all the problems" pluggable text renderer.

Reply:
I WANT IT NOW! We should throw in kubernetes too somehow ;)

Reply:
@katajakasa Would this be something you could rework later, after this change goes in?

Reply:
Yes, it is. Let's get this part merged first.